GitVersion.yml

```
@@ -0,0 +1,8 @@
+mode: ContinuousDelivery
+assembly-file-versioning-format: '{Major}.{Minor}.{Patch}.{CommitsSinceVersionSource}'
+assembly-informational-format: '{Major}.{Minor}.{Patch}.{CommitsSinceVersionSource}-{BranchName}+{ShortSha}'
+merge-message-formats: {}
+commit-message-incrementing: Enabled
+patch-version-bump-message: "^(add|bump|drop|mv|move|update|expose|remove|refactoring|additional|use|improvement|fix|refactor)(.+)?"
+tag-prefix: 'IGNORE'
+next-version: "0.1"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)